wazimap_ng/urls.py

```diff
@@ -149,6 +149,11 @@ def trigger_error(request):
         name="all-details"
     ),
 
+    path(
+        "api/v1/indicator_data/profile/<int:profile_id>/geography/<str:geography_code>/",
```

Comment:
does this return all the data for a single geography_code? How is it different to all-details?

Reply:
Not specified in url but format should be `Geography_code:version1:version2:version3`
Will look for proper way to make this type of url.
In backend we are splitting geography_code by `:` and get list of versions and geography code

Reply:
Probably wanna keep this out of the first iteration on this